app/javascript/src/Navigation/components/header/Toolbar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Toolbar as PFToolbar, ToolbarGroup, ToolbarItem } from '@patternfly/react-core'
+import AccountSettings from 'Navigation/components/header/AccountSettings'
+import DocumentationMenu from 'Navigation/components/header/DocumentationMenu'
+
+class Toolbar extends React.Component {
```

Comment:
I'd use a stateless component here. A function returning the JSX will do better. This is using only props